progenitor-impl/src/method.rs

```diff
@@ -138,6 +142,7 @@ pub enum BodyContentType {
     Json,
     FormUrlencoded,
     Text(String),
+    FormData(bool),
```

Comment:
would be good to add doc here to indicate that the `bool` (iiuc) holds whether it is required or not.  It feels like this is a strange spot to store that info.

Reply:
Documented it. 
It's the least surprising spot, since it follows the convention of parent variants in `OperationParameterKind` having such a member. That is, until general optionality of body parameters is solved (not in this PR).